index.md

```diff
@@ -8,47 +8,47 @@
 
 ### <center>  ١-النقد</center>
 
-<center> نحن نؤمن بأهمية معرفة وتقبل النقد من الشركة بثقة وسلاسة مهما كان قاسيا والعمل على تطبيقه والاستفادة منه لاقصي درجة لأنه بمثابة الوسيلة الأساسية للنمو والنضوج داخل الشركة والمصدر الرئيسى للتطوير واكتساب مهارات جديدة ومختلفة في بيئة العمل</center>
+<center> نحن نؤمن بأهمية معرفة وتقبل النقد من الشركة بثقة وسلاسة مهما كان قاسيا والعمل على تطبيقه والإستفادة منه لأقصي درجة بل و طلب التغذية المرتدة ايضا لأنها بمثابة الوسيلة الأساسية للنمو والنضوج داخل الشركة والمصدر الرئيسى للتطوير واكتساب مهارات جديدة ومختلفة في بيئة العمل.</center>
```

Comment:
I adjust the statement to be:

نحن نؤمن بأهمية طلب التغذية العكسية ومعرفة وتقبل النقد من الشركة بثقة وسلاسة مهما كان قاسيا والعمل على تطبيقه والإستفادة منه لأقصي درجة لأنه بمثابة الوسيلة الأساسية للنمو والنضوج داخل الشركة والمصدر الرئيسى للتطوير واكتساب مهارات جديدة ومختلفة في بيئة العمل

Also, please remove the dot (.) because as you can see it appears in the wrong alignment (right) unless you can fix that to appear in the left side?